index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+
+</head>
+<body>
+    <header class="toolbar">
+        <a href="(##" class="name"> SOLOMON</a>
+        <nav>
+          <img class="huberger" src="/assets/Union.png" alt="Dropdown menu">
+        </nav>
+    </header>    
+     <section class="first-page">
+           <div class="headline">
+             <h1 class="hey">Hey There  I'm Solomon</h1>
+             <h2 class="softwaredev"> I am software developer</h2>
+             <p class="headline-p">I can help you build a product , feature or website Look through some of my work <br>
+                 and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+             </p>
+          </div>
+          <ul class="socila-link">
+            <li class="social"><a href="##"><img  src="/assets/socialmedia/github.png" alt="Github"></a></li>
+            <li class="social"><a href="##"><img  src="/assets/socialmedia/linkidin.png" alt="linkedin"></a></li>
+            <li class="social"><a href="##"><img  src="/assets/socialmedia/tump.png" alt="tum"></a></li>
+            <li class="social"><a href="##"><img  src="/assets/socialmedia/tiwtter.png" alt="tiwtter"></a></li>
+            <li class="social"><a href="##"><img  src="/assets/socialmedia/m.png" alt="M"></a></li>
```

Comment:
- I can see that you have made great efforts here but your images are not showing now. It looks like your file path is missing a `.` before `/assets/...`. It's even possible that the images are visible on your end, and that may be due to your cache memory.
- [ ] Can you try this 👇  path instead?
```
<li class="social"><a href="profile_link"><img  src="./assets/socialmedia/github.png" alt="Github"></a></li>
            <li class="social"><a href="profile_link"><img  src="./assets/socialmedia/linkidin.png" alt="linkedin"></a></li>
            <li class="social"><a href="profile_link"><img  src="./assets/socialmedia/tump.png" alt="tum"></a></li>
            <li class="social"><a href="profile_link"><img  src="./assets/socialmedia/tiwtter.png" alt="tiwtter"></a></li>
            <li class="social"><a href="profile_link"><img  src="./assets/socialmedia/m.png" alt="M"></a></li>
```
- [ ] Kindly remember to change the value of your `href` from `profile_link` to your exact profile page so that users can connect or contact you whenever they click on it. This is good for user experience.
- [ ] [OPTIONAL] In case you want your site to be very accessible to screen readers or on the event of server issues/internet issues, consider making your `alt` text more descriptive i.e. tell what the image is about.